### DISCUSSION TITLE
sqlmap integrated DefectDojo

### DISCUSSION THREAD

**Discussion by Bank4500** [DefectDojo/django-DefectDojo#4970] [Q&A]
sqlmap integrated DefectDojo can be combined? what to do

**Comment by damiencarol** [comment]
Do you mind giving more details about what "sqlmap" is?

**Reply by valentijnscholten** [reply]
https://sqlmap.org/

**Reply by valentijnscholten** [reply]
You can import sqlmap reports via the API or UI just like any other scanner. Running sqlmap periodically is outside the scope of Defect Dojo.
